w07_policy_search/t06_pg_algorithms.tex

```diff
@@ -94,11 +94,11 @@
 	
 	$$\nabla_\theta \mathbb{E}_\tau [R] = \mathbb{E}_\tau \left[ \sum_{t=0}^{T-1} \nabla_\theta \log \pi(a_t \mid s_t; \theta) \left( \sum_{t'=t}^{T-1} r_{t'} - b(s_t) \right) \right] $$
 	
-	\item For any choise of $b$, gradient estimator is unbiased
+	\item For any choice of $b$, gradient estimator is unbiased
 	\item Near optimal choice is the expected return
 	$$b(s_t) \approx \mathbb{E} [r_t + r_{t+1} +\ldots + t_{T-1}] $$
 	
-	\item Interpretation: increase logprob of action $a_t$ proportionally to how much returns $\sum_{t'=t}^{T-1} r_{t'}$ are better than expected
+	\item Interpretation: increase logprob of action $a_t$ proportionally to how many returns $\sum_{t'=t}^{T-1} r_{t'}$ are better than expected
```

Comment:
I think we want much here, which measures the amount of reward the exceeds the expectation vs many which measures the number of steps in which the reward is better